dashboard/app/models/script.rb

```diff
@@ -1072,11 +1072,8 @@ def self.setup(custom_files, new_suffix: nil, new_properties: {}, show_progress:
     progressbar = ProgressBar.create(total: units_to_add.length, format: '%t (%c/%C): |%B|') if show_progress
 
     # Stable sort by ID then add each unit, ensuring units with no ID end up at the end
-    added_unit_names = units_to_add.sort_by.with_index {|args, idx| [args[0][:id] || Float::INFINITY, idx]}.map do |options, raw_lesson_groups|
-      added_unit =
-        options[:properties][:is_migrated] == true ?
-          seed_from_json_file(options[:name]) :
-          add_unit(options, raw_lesson_groups, new_suffix: new_suffix, editor_experiment: new_properties[:editor_experiment])
+    added_unit_names = units_to_add.sort_by.with_index {|args, idx| [args[0][:id] || Float::INFINITY, idx]}.map do |options, _raw_lesson_groups|
+      added_unit = seed_from_json_file(options[:name])
```

Comment:
this code path is dead now, but it seemed better to make this one change to it to make it super clear that `add_unit` can no longer be called from here.